### ISSUE TITLE
Are there any plans for 1.20.1?

### ISSUE THREAD

**Issue by catreplicators** [LubieKakao1212/OpenCombatUtility#9]
This mod seems quite interesting.

**Comment by LubieKakao1212** [comment]
Yes, there will be a 1.20.1 version for both fabric and forge/neoforge
There is also a roadmap on [Discord](https://discord.com/invite/3dDJwBBYTk)
